src/collections/pages/en/projects.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Projects
+permalink: en/projects/
```

Comment:
Also worth noting that the home page in English doesn't have the `en` prefix so this probably shouldn't either.